my_gazebo_concert/src/solutions/gazebo.services

```diff
@@ -0,0 +1,13 @@
+# PRIORITIES
+#
+# Turtle pond has scheduler_msgs.Request.DEFAULT_PRIORITY, teleop has 
+# scheduler_msgs.Request.HIGH_PRIORITY. If you want to change the priority
+# override it as below for any of the services:
+#  overrides:
+#    priority: 1000
+
+- resource_name: concert_service_gazebo/gazebo_robot_world
+  overrides:
+    parameters: gazebo_concert/gazebo.parameters
```

Comment:
You have already copied the file. You just need to edit this file: https://github.com/tfoote/turtlebot/pull/4/files#diff-f962c21ad1a9b0855a2ebcd8ec95efffR1